### ISSUE TITLE
Addition of WTA Data

### ISSUE THREAD

**Issue by pipolopes5** [mcekovic/tennis-crystal-ball#289]
Hi, 

Do you have any plans in the near future to add WTA data? And, if possible, in the same type of format?

Really like the website, is very helpfull! It would be even better with female tennis stats!

Thanks

**Comment by mrsethallen** [comment]
I agree that the WTA stuff would be great but I read in previous issues that it really wasn't a priority. I'm currently learning to code and am grateful that he updates this every few days. I'm not sure how much work is involved week to week but if I could help in anyway I'd be willing to devote time. As an analyst its a very good resource.

**Comment by mcekovic** [comment]
Re WTA: It is a lot of work (currently do not have that much free time). WTA data from Jeff repo needs to be loaded and cleaned up, tournament classification for woman tennis needs to be designed, data scrappers for updates of new data written (for new tournaments, new rankings and in progress tournaments), web site needs to support dual interface... simply too much work.
Re updates: they are automatic, on Monday mornings, but I need to check if everything went ok and sometimes things go wrong, so I need to correct. About every year, in the off season, the DB is rebuild from scratch. In general, reasonable amount of work to keep the data up to date, not too much. But initial amount of work for WTA is currently too much for me.

**Comment by bernardobraga92** [comment]
@mcekovic thanks so much for your reply and I have to say I appreciate a lot that you made this site possible even tho it gives you a lot of work. Currently me and @pipolopes5 are developing some stuff that's why he asked about WTA stats. Maybe in the future we can take on your code and make the same to the female circuit!

**Comment by bmurtagh** [comment]
Definitely in favor of this request.
